src/include/function/string/operations/reg_expr_operation.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "common/types/ku_string.h"
+#include "function/string/operations/find_operation.h"
+#include "re2.h"
+
+using namespace kuzu::common;
+using namespace kuzu::regex;
+
+namespace kuzu {
+namespace function {
+namespace operation {
+
+struct RegExpr {
```

Comment:
The general suggestion is to not call the operation/function as RegExpr, instead be more specific, this is a regular expression full match function. let's rename them follow the name "RegExprMatch" or "REMatch".

Reply:
done